modern/src/reports/SummaryReportPage.js

```diff
@@ -99,8 +98,8 @@ const SummaryReportPage = () => {
             </TableRow>
           </TableHead>
           <TableBody>
-            {items.map((item, index) => (
-              <TableRow key={index}>
+            {items.map((item) => (
+              <TableRow key={item.deviceId}>
```

Comment:
This probably won't work for daily report.

Reply:
I want to ask you to check that all the key I used are unique one, if deviceId not a unique key for the SummaryReport, I'm just using the demo generated by test-generator, and I got only one row as result on summary Report I couldn't find if deviceId is a unique key or not. there is also a console.log that I forgot on Statistics Report

Reply:
Can you combine device id + time as a key here?

Reply:
I combined as you suggested deviceId with startTime (ms) and separated them with ' _ '